feature-libs/user/profile/components/register/register-form.service.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { UntypedFormArray, UntypedFormBuilder } from '@angular/forms';
+import { ConsentTemplate } from '@spartacus/core';
+
+@Injectable()
+export class RegisterFormService {
+  constructor(protected fb: UntypedFormBuilder) {}
+
+  loadExtraRegistrationConsents(): {
```

Comment:
Make the name of the method as 'loadAdditionalConsents'

Reply:
done

Reply:
Just asking: why do we even bother with the word "additional"? Can't be simply "loadConsents"? (and analogically `generateConsentsFormControl`)?